src/dlgprefvinyldlg.ui

```diff
@@ -593,65 +593,65 @@
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
-       <layout class="QHBoxLayout" name="horizontalLayout_2">
+       <widget class="QLabel" name="textLabel1_3">
+        <property name="text">
+         <string>Turntable input signal boost</string>
```

Comment:
sorry, actually I should have asked for the to be Title Capitalization: Turntable Input Signal Boost